### ISSUE TITLE
Home row layer instead of mod

### ISSUE THREAD

**Issue by arabshapt** [urob/zmk-config#21]
Hi there, I am using home row mods on other alpha keys as well. But I was wondering whether it is possible to use the advancements in this feature(differenciating between tap and hold) for a layer switch. Example: holding j would switch layer instead of making e.g. hold CMD/GUI. 
I really appreciate what you are doing. Thank you for these repos and the guide how to use git for zmk.

**Comment by urob** [comment]
Yes, you want to replace the first binding in the behavior definition from `kp` to `mo`.

**Comment by arabshapt** [comment]
Thank you) it worked:)
